packages/components/src/utils/reduce-motion.js

```diff
@@ -3,6 +3,10 @@
  *
  * @param {'transition' | 'animation' | string} [prop='transition'] CSS Property name
  * @return {string} Generated CSS code for the reduced style
+ *
+ * @deprecated Write your own media query instead,
+ * e.g. `@media not ( prefers-reduced-motion ) { ...some animation... }` or
```

Comment:
Maybe this is just a typo or maybe it's another valid way to implement this, but:

```diff
-@media not ( prefers-reduced-motion ) {
+@media not ( prefers-reduced-motion: reduce ) {
```

Are we sure those will behave the same way with no weird edge cases? Is there any practical difference between these two that could be relevant?

Reply:
The [current docs](https://drafts.csswg.org/mediaqueries-5/#mq-boolean-context) do say they should be equivalent. They would differ if they ever added another possible value for [`prefers-reduced-motion`](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion), for example `reduce-somewhat` or `absolutely-no-motion-im-serious`, but in that case I think it'd be better for those values to be included in our simple booleans by default.

Reply:
The linked spec section is a very interesting read, and makes a lot of sense 👍